.commitlintrc

```diff
@@ -9,7 +9,8 @@
 				'web',
 				'client-http',
 				'server',
-				'react'
+				'react',
```

Comment:
This is weird? Didn't we merge these changes yesterday with the react example?

Reply:
It's also in #24?

Reply:
rebased